specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -1845,6 +1845,12 @@
           }
         ],
         "responses": {
+          "200": {
+            "description": "On static validation errors for validateOnly.",
```

Comment:
Existing description "When validateOnly is true this will run validations and return warnings and errors if any, hostpool will not be updated." of corresponding "ValidateOnly" flag is not clear, please ask Shaila to update the description.

I expected the description will clearly describe when it is true, it will run static validations (provides examples) and long running validation; if set to false, only static validation and save/schedule host pool update.

Reply:
updated to: "On static validation errors for validateOnly equals true."